Riot/Categories/UILabel.swift

```diff
@@ -42,34 +42,11 @@ extension UILabel {
     /// Sets an HTML string into the receiver. Does not support custom fonts but considers receiver's font size.
     /// - Parameter htmlText: HTML text to be rendered.
     @objc func setHTMLFromString(_ htmlText: String) {
-        let htmlTemplate = """
-        <!doctype html>
-        <html>
-          <head>
-            <style>
-              body {
-                font-family: -apple-system;
-                font-size: \(font.pointSize)px;
-              }
-              p:last-child { display: inline; }
-            </style>
-          </head>
-          <body>
-            \(htmlText)
-          </body>
-        </html>
-        """
+        let html = "<!doctype html><html><head><style>body { font-family: -apple-system; font-size: \(font.pointSize)px; } p:last-child { display: inline; }</style></head><body>\(htmlText)</body></html>"
```

Comment:
Does removing spaces make a difference? I believe this was more readable before.

Reply:
I guess it doesn't matter now that I will remove the style, the issue was that DTCoreText would include some leading spaces, I did that rather than trimming. (but to be fair I should've done only ```<body>\(htmlText)</body>```, it's probably enough)